data.sql

```diff
@@ -1,8 +1,20 @@
 /* Populate database with sample data. */
 
+-- Day1
+
 INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg)
 	VALUES 	('Agumon', '2020-3-2', 0, FALSE, 10.23),
 			('Gabumon', '2018-11-15', 2, TRUE,8.00),
 			('Pikachu', '2021-1-7', 1, FALSE,15.04),
 	 		('Devimon', '2017-5-12', 5, TRUE,11.00);
 
+
+-- DAY 2
+INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg)
+	VALUES 	('Charmander', '2020-2-8', 0, FALSE ,-11),
+			('Plantmon', '2020-11-15', 2, TRUE ,-5.7),
```

Comment:
- [ ] Kindly fix the born Year for Plantmon. It should be 2021 instead of 2020 :+1: 

**Project requirement:** Animal: Her name is Plantmon. She was born on Nov 15th, 2021, and currently weighs -5.7kg. She is neutered and she has tried to escape 2 times.